src/Sql/Sql/ServiceObjective/Cmdlet/GetAzureSqlServerServiceObjective.cs

```diff
@@ -43,12 +44,12 @@ protected override IEnumerable<AzureSqlServerServiceObjectiveModel> GetEntity()
         {
             ICollection<AzureSqlServerServiceObjectiveModel> results = null;
 
-            if (this.MyInvocation.BoundParameters.ContainsKey("ServiceObjectiveName") && !WildcardPattern.ContainsWildcardCharacters(ServiceObjectiveName))
+            if (this.MyInvocation.BoundParameters.ContainsKey("ServiceObjectiveName"))
```

Comment:
Don't remove this - this is required for wildcards to work.

Reply:
Oops, this was a bad merge. Thank you for spotting this.